packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -317,6 +318,25 @@ export const FLAGS: Record<string, FlagInfo> = {
     compatibilityWithOldBehaviorMd: 'You can pass `desiredCount: 1` explicitly, but you should never need this.',
   },
 
+  [ECS_PATTERNS_SMART_DEFAULT_OPEN_LISTENER]: {
```

Comment:
A default that changes isn't really a default, and "smart" isn't a descriptive name. I would suggest to rename this to something more descriptive, like `ECS_PATTERNS_SEC_GROUPS_DISABLES_IMPLICIT_OPEN_LISTENER`